examples/pandapipes/input/timeseries_constant _original.csv

```diff
@@ -0,0 +1,46 @@
+DateTime,demand_1
```

Comment:
I think this file is not used by any of the runs, can we remove it?

Reply:
Thanks. Deleted